models/_sector/dex/trades/arbitrum/_schema.yml

```diff
@@ -15,5 +15,16 @@ models:
           combination_of_columns:
             - tx_hash
             - evt_index
+      - check_seed:
+          seed_file: ref('uniswap_arbitrum_base_trades')
+          match_columns:
+            - tx_hash
+            - evt_index
+          check_columns:
+            - block_number
```

Comment:
i believe `block_number` should be in the `match_columns` list, not `check_columns`, as we'd intend for it to be part of the join condition instead, right? that's my read on it. i guess technically either way works, as the two current match columns are unique, so adding `block_number` there doesn't add much value. i guess i'm good with how you have it now.

Reply:
sorry that was confusing, i started off saying to change, then decided it's fine 🤣